src/components/training/TrainingPlanOverview.jsx

```
@@ -0,0 +1,1120 @@
+                          : isActiveWorkout
+                          ? "border-blue-500/50 shadow-blue-500/10 hover:shadow-blue-500/25 hover:border-blue-400/70"
+                          : isRestDay
+                          ? "border-purple-500/40 shadow-purple-500/10 hover:shadow-purple-500/25 hover:border-purple-400/60"
+                          : "border-zinc-700/50 hover:border-zinc-600/70 hover:shadow-zinc-600/10"
+                      }`}
+                      onClick={(e) => {
+                        if (e.target === e.currentTarget || e.target.closest(".day-main-content")) {
+                          toggleDayExpansion(dayIdx);
+                        }
+                      }}
+                    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)